user/src/main/java/com/user/outDto/UserOutDto.java

```diff
@@ -0,0 +1,12 @@
+package com.user.outDto;
```

Comment:
Folder name must be all in lowercase, like not outDto it should be indto

Reply:
done